### PR TITLE
fix: missing trailing `---` in formatting guide

### PR DESCRIPTION
**Pull request by kennethkalmer**
Spotted by @fliptopbox right after merging #1016.

Although this didn't cause a rendering issue on docs.ably.io, it might on the website. However, the formatting guide isn't directly accessible from the website navigation.

### CODE REVIEW

File: content/client-lib-development-guide/documentation-formatting-guide.textile
Comment (tbedford) [inline content/client-lib-development-guide/documentation-formatting-guide.textile:62]: There seems to be an errant space on the front of this.

**Review by tbedford — CHANGES_REQUESTED** [review]

File: content/client-lib-development-guide/documentation-formatting-guide.textile
Comment (kennethkalmer) [reply content/client-lib-development-guide/documentation-formatting-guide.textile:62]: I copied it from other lines earlier in the file because I wasn't sure on the exact syntax. Should I fix all the spacing in another commit to be consistent?

**Review by tbedford — APPROVED** [review]
LGTM

**Comment by tbedford** [comment]
@kennethkalmer do approvers normally merge, or leave for assignee to merge?

**Comment by kennethkalmer** [comment]
> @kennethkalmer do approvers normally merge, or leave for assignee to merge?

@tbedford depends on the team that own the repo. For repos the website team owns we take responsibility for merging contributions and getting the deployments done and we assume the same for others unless they tell us we can safely merge after approval.

You let us know what flow you want for docs and we'll follow along :)

**Comment by tbedford** [comment]
@kennethkalmer - my preference would be:

1. Creator of the PR "owns" the PR.
2. All docs repo PRs should be reviewed by at least one member of the technical writing team.
2. The owner of the PR merges it once they have at least one approval. They manage the PR as they see fit.
3. Exception is external contributors who don't have privilege to merge. In that case once approved a member of the docs team can merge.

So for example, in this case you have an approval (from a member of the docs team) so you can merge if you want. If you need further review or an additional approval etc. then that is under your control.

**Pull request merged**
